### DISCUSSION TITLE
Postal stop sending email

### DISCUSSION THREAD

**Discussion by jossybenson** [postalserver/postal#1661] [Help with using Postal]
Hello I have only postal Install on my digital ocean server, everything is working fine but since yesterday it stopped sending emails, what ever mail sender software I use it still don't work. Please I need your help.

**Comment by willpower232** [comment]
what is the output of `postal status`?

**Comment by jossybenson** [comment]
My postal status everything is up, that is 
Postal_cron_1
Postal_requeuer_1
Postal_smtp_1
Postal_web_1
Postal_worker_1 all the status is up

**Reply by willpower232** [reply]
so when you say it has stopped sending emails, what is happening? Can you see them in Postal or can you not get the messages to Postal?

**Reply by jossybenson** [reply]
When sending messages it always send back error message like connection error can not connect with your server, no matter the kind of mail sender software I use. And this don't happen before....

**Reply by willpower232** [reply]
can you share a screenshot of the error?

**Reply by jossybenson** [reply]
Ok I will share with you just a moment please

**Reply by jossybenson** [reply]
![Capture](https://user-images.githubusercontent.com/92524219/139236661-78365aad-a279-4b57-965b-2f9a0f07ba5b.PNG)

**Reply by jossybenson** [reply]
even if I use Tls or SSL still thesame thing, it will not send mail to yahoo, gmail, hotmail and any other mail.

**Reply by pjv** [reply]
I think the OP mentioned somewhere that they set up their postal server on a digital ocean VPS. Something like 80% (ok, i pulled that number out of thin air, but i have tried it and it's a large %) of the Internet will not accept email from an IP address in the Digital Ocean block. It doesn't matter what the history of the particular IP address is, the entire DO block is listed on many, many RBLs.

So here's a guess that Postal is working fine, but no mail is delivered because the OP's traffic is being blocked by recipients.

**Reply by jossybenson** [reply]
Ok pjv that is to say the fault is not from me right?

**Reply by pjv** [reply]
No, that is to say that my guess is the reason you can't send email from your postal install is because it is hosted on a digital ocean droplet. That's a **guess**. If you want to know for sure what the actual problem is, you have to dig into the details. For instance, the screenshot you showed above is non-informative. But it might be almost telling you something. The cut-off red text in the little box in the upper right that starts with the word "Error" might tell you something useful if you look at the whole error message. 

Nobody can help you debug "it's not working". You have to dig into exactly HOW it's not working.

**Reply by jossybenson** [reply]
ok thank you so much! I really appreciate.

**Reply by jossybenson** [reply]
hello @pjv why if I send a test mail from smtper.net to yahoo and gmail it work fine?

**Reply by pjv** [reply]
i have no idea whatsoever what you are asking. Sorry.

**Reply by jossybenson** [reply]
ok no problem, I decided to use a mail sender online to test my mail server delivery status and I am surprise to see https://smtper.net/ send and deliver my mails to both yahoo and gmail.

**Reply by willpower232** [reply]
if it works from smtper.net and not from your software then there must be a problem local to you. Is it possible you have a firewall or something on your internet connection?

It would be nice if "Super Email Sender" was able to tell you what exactly was wrong, I am not familiar with that software to suggest where to look.

**Reply by jossybenson** [reply]
Ok willpower super mail sender said error A connection attempt fail because the connection.......it never complete the message, if it is firewall how can I look for it and stop it from blocking my mails?

**Reply by jossybenson** [reply]
@pjv @willpower232 I now use Morphy mail sender and it was successful, now is working just fine! I still want to thank you guys for helping me......I really appreciate thanks guys.

**Reply by Drains069** [reply]
im also going through this issue please help me with the Morphy mail sender
